README.md

```diff
@@ -28,6 +44,195 @@ The Cosmos DB Data Migration Tool core project is a C# command-line executable.
+
+    >**Note**: When using the Cosmos DB Data Migration tool, the container doesn't have to previously exist, it will be created automatically using the partition key specified in the sink configuration.
+
+### Task 2: Prepare JSON source documents
+
+1. Locate the **docs/resources/sample-data.zip** file. Extract the files to any desired folder. These files serve as the JSON data that is to be migrated to Cosmos DB.
+
+### Task 3: Setup the data migration configuration
+
+1. Each extension contains a README document that outlines configuration for the data migration. In this case, locate the configuration for [JSON (Source)](https://github.com/Azure/azure-documentdb-datamigrationtool/tree/feature/begin-core/Extensions/Json/Cosmos.DataTransfer.JsonExtension) and [Cosmos DB (Sink)](https://github.com/Azure/azure-documentdb-datamigrationtool/tree/feature/begin-core/Extensions/Cosmos/Cosmos.DataTransfer.CosmosExtension).
+
+2. In the Visual Studio Solution Explorer, expand the **Microsoft.Data.Transfer.Core** project, and open **migrationsettings.json**. This file provides an example outline of the settings file structure. Using the documentation linked above, configure the **JsonSourceSettings** and **Cosmos-NoSqlSinkSettings** sections. Ensure the **FilePath** setting is the location where the sample data is extracted. The ConnectionString setting can be found on the Cosmos DB Emulator **Quickstart** screen as the **Primary Connection String**. Save the file.
+
+    ```json
+    {
+        "SourceSettings": {            
+            "FilePath": "C:\\btcdata\\simple_json.json"
+        },
+        "SinkSettings": {
+            "ConnectionString": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDj...",
+            "Database": "datamigration",
+            "Container": "btcdata",
+            "PartitionKeyPath": "/id",
+            "RecreateContainer": false,
+            "IncludeMetadataFields": false
+        }
+    }
+    ```
+
+    ![The Cosmos DB Emulator Quickstart screen displays with the Primary Connection String value highlighted.](media/cosmosemulator_primaryconnectionstring.png "Cosmos DB Emulator Primary Connection String")
+
+3. Ensure the **Cosmos.DataTransfer.Core** project is set as the startup project then press <kbd>F5</kbd> to run the application.
+
+4. The application will load the provided extensions, then prompt for a **Source**. Select **JSON**. When prompted to load settings from a file enter `n` and press <kbd>Enter</kbd>. The application provides the ability to configure extensions in a separate file versus having them located in the **migrationsettings.json** file. When a separate file is not specified, the settings are read from **migrationsettings.json**. When prompted to provide the Configuration section name, press <kbd>Enter</kbd> to accept the default: **JSONSourceSettings**.
```

Comment:
The config name step has been cut out so the last sentence can be removed. The y/n step has also been cut down to just ask for a file path and use migrationsettings if nothing is entered.

Reply:
Updated